client/src/store/new-request-module.ts

```diff
@@ -2307,10 +2311,10 @@ export class NewRequestModule extends VuexModule {
         data['corpNum'] = this.corpNum
       }
 
-      const requestData = await this.addRequestActionComment(data)
+      const requestData = data && await this.addRequestActionComment(data)
 
       try {
-        const response = await axios.put(`/namerequests/${nrId}`, requestData, {
+        const response = requestData && await axios.put(`/namerequests/${nrId}`, requestData, {
```

Comment:
One more.

I guess this will cause a JS "undefined" error that will be caught by the catch, but this isn's ideal. Would be better to handle this scenario with an appropriate error (or log) and return false, or something along those lines.

Reply:
This is indeed the case, as with all of them. They are caught when the request doesn't occur and the error is boiled up from the point of failure. 
This still needs work for sure, to flesh out and isolate these (on top of fixing the individual errors if we can)

Reply:
OK, create a ticket or add to the description of an exsting ticket to track this work. Thanks!